05_stringer/mohitnag/main_test.go

```diff
@@ -0,0 +1,39 @@
+package main
+
+import (
+	"bytes"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestIpaddr(t *testing.T) {
+	testData := []struct {
+		Scenario string
+		input    IPAddr
+		expected string
+	}{
+		{"Scenario One", [4]byte{127, 0, 0, 1}, "127.0.0.1"},
+		{"Scenario two", [4]byte{}, "0.0.0.0"},
+		{"Scenario three", [4]byte{0, 0, 0, 10}, "0.0.0.10"},
```

Comment:
Would be better to test edge cases like 255.255.255.255

Reply:
Test the case like [4]byte{10}

Reply:
Done